### ISSUE TITLE
Copying/moving images with `source` and `target` corrupts them

### ISSUE THREAD

**Issue by halhenke** [lukeed/taskr#72]
A flyfile function

``` javascript
export function* images () {
  yield this
    .source(paths.images)
    .target("public/images")
}
```

copies images from `paths.images` to `public/images` but the resulting files have the wrong file size, no dimensions etc. Dont think i've bungled the syntax as it works fine for other file types.

**Comment by ghost** [comment]
This is definitely an encoding bug, I made the silly mistake of always using `utf8`. 

`Fly.proto.target` should allow a way to specify the encoding and also, Fly could try checking the extension for common file types.

**Comment by halhenke** [comment]
Ah ok - the line

`this.encoding = process.env.ENCODING || "utf8"`

I guess there is probably a library or some code for checking file types with extensions floating around somewhere? - might take a quick look and see if I find anything useful.

**Comment by ghost** [comment]
@halhenke Yes. Sounds good. :+1: 

I added this to the next milestone, shipping this week :smile:

**Comment by ghost** [comment]
Fixes in 0.6.0 over the weekend.

**Comment by ghost** [comment]
Done! :smile:
